### DISCUSSION TITLE
提案：記事の執筆ページにおいてIME変換中はプレビューを更新しないように

### DISCUSSION THREAD

**Discussion by yamadashy** [increments/qiita-discussions#135] [要望・アイデア・不具合]
すごく細かい部分の提案で申し訳ないのですが、

記事の執筆中リアルタイムにプレビューが反映されますが、更新のタイミングがまばらなのもありIME変換中に更新されると「これによっｔ」のような中途半端な状態になります。

JS でそのあたりのイベントは取れると思うので、お手すきで対応いただけると幸いです。

**Comment by kyntk** [comment]
@yamadashy ありがとうございます!
私自身書いていて日頃気になる部分なので、ぜひやりたいと思っています。

すぐには対応方法がわからないので、実現方法の調査をしてみます 🙇🏽

**Comment by yamadashy** [comment]
ご返信ありがとうございます。

このあたりのイベントを捕捉するイメージでございます。
https://developer.mozilla.org/ja/docs/Web/API/Element/compositionstart_event

お手数ですがよろしくお願いいたします :bow:

**Comment by tsuzuki-takaaki** [comment]
@yamadashy
報告が遅くなってしまい、申し訳ございません。

こちらの不具合ですが、現在は修正が完了しております！
記事の執筆中に、中途半端な状態でプレビューが更新される挙動は解消されているはずですので、ご確認いただけますと幸いです。

下記の discussion でも本件に近い内容に触れておりますので、合わせてご確認いただけますと幸いです。
https://github.com/increments/qiita-discussions/discussions/996

よろしくお願いいたします🙇‍♂️

**Reply by yamadashy** [reply]
ご対応いただきありがとうございます！

後ほど確認させていただきます

**Reply by yamadashy** [reply]
@tsuzuki-takaaki
修正されていることを確認いたしました。ありがとうございます
closeします！
